build-farm/set-platform-specific-configurations.sh

```diff
@@ -24,8 +24,8 @@ then
     export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-warnings-as-errors"
 fi
 
-# jdk-19 and above support reproducible builds
-if [[ "${JAVA_FEATURE_VERSION}" -ge 19 ]]
+# jdk-17 and above support reproducible builds
+if [[ "${JAVA_FEATURE_VERSION}" -ge 17 ]]
```

Comment:
Were the changes backported to 18?

Reply:
Right! I misunderstood the requirement, thought it meant to be 17+ but actually only 17